src/main/resources/view/MainWindow.fxml

```diff
@@ -37,30 +31,27 @@
           </Menu>
         </MenuBar>
 
-        <StackPane VBox.vgrow="NEVER" fx:id="commandBoxPlaceholder" styleClass="pane-with-border">
+        <StackPane fx:id="commandBoxPlaceholder" styleClass="pane-with-border" VBox.vgrow="NEVER">
           <padding>
-            <Insets top="5" right="10" bottom="5" left="10" />
+            <Insets bottom="5" left="10" right="10" top="5" />
```

Comment:
Was there a reason for swapping the order of 'positions' to `top="5" right="10" bottom="5" left="10"`?

I ask this because the rest of the `.fxml` files seem to use the order of `top`, `right`, `bottom`, `left`.